### PR TITLE
[8/n][pipeline-gen] Add method to generate build step

### PR DESCRIPTION
**Pull request by khluu**
- Add method to generate build step based on commit, container registry and image.

### CODE REVIEW

**Comment by aslonnie** [comment]
is this for review? could you add a PR description maybe?

File: scripts/pipeline_generator/pipeline_generator_helper.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+from typing import List
+
+def get_build_commands(container_registry: str, buildkite_commit: str, container_image: str) -> List[str]:
```
Comment (aslonnie) [inline scripts/pipeline_generator/pipeline_generator_helper.py:3]: any unit test? or is this so simple that does not require one?

maybe we should have some form of integration test to test these commands.

Comment (khluu) [reply scripts/pipeline_generator/pipeline_generator_helper.py:3]: It's just replacing placeholders for container registry, container image, and commit so I don't think unit test is needed here.

File: scripts/pipeline_generator/pipeline_generator_helper.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+fi
+"""
+    docker_build_command = (
+        f"docker build "
+        f"--build-arg max_jobs=64 "
+        f"--build-arg buildkite_commit={buildkite_commit} "
+        f"--build-arg USE_SCCACHE=1 "
+        f"--tag {container_image} "
+        f"--target test "
+        f"--progress plain ."
```
Comment (aslonnie) [inline scripts/pipeline_generator/pipeline_generator_helper.py:23]: could you leave a todo here to not using `.` to include everything as the build context?

**Review by aslonnie — APPROVED** [review]

**Pull request merged**